### ISSUE TITLE
Analysis of almost all Nuget packages fails

### ISSUE THREAD

**Issue by peteawood** [aws/porting-assistant-dotnet-datastore#5]
When running an assessment I don't get any compatibility score and almost all of my Nuget dependencies are marked with a status of "Failed" (not "Incompatible").

I have pre-built the solution successfully and I don't get any build issues with the tool. Some of the nuget packages are from a private repo but the majority are public and have newer, compatible versions (for instance CSVHelper and FakeItEasy).

In my assessment log I have a number of errors similar to the following:

```
[2020-08-25 10:49:15 INF] EncoreCache.ExternalPackagesCompatibilityChecker: Downloading fakeiteasy.json.gz from aws.portingassistant.dotnet.datastore
[2020-08-25 10:49:16 ERR] EncoreCache.ExternalPackagesCompatibilityChecker: Failed when download and parsing fakeiteasy.json.gz from aws.portingassistant.dotnet.datastore, Amazon.S3.AmazonS3Exception: Access Denied
 ---> Amazon.Runtime.Internal.HttpErrorResponseException: Exception of type 'Amazon.Runtime.Internal.HttpErrorResponseException' was thrown.
   at Amazon.Runtime.HttpWebRequestMessage.GetResponseAsync(CancellationToken cancellationToken)
....
[2020-08-25 10:49:33 INF] EncoreCache.ExternalPackagesCompatibilityChecker: Downloading system.json.gz from aws.portingassistant.dotnet.datastore
[2020-08-25 10:49:33 ERR] EncoreCache.ExternalPackagesCompatibilityChecker: Failed when download and parsing system.json.gz from aws.portingassistant.dotnet.datastore, Amazon.S3.AmazonS3Exception: Access Denied
 ---> Amazon.Runtime.Internal.HttpErrorResponseException: Exception of type 'Amazon.Runtime.Internal.HttpErrorResponseException' was thrown.
   at Amazon.Runtime.HttpWebRequestMessage.GetResponseAsync(CancellationToken cancellationToken)
```

The first line of the stack trace for these errors is `EncoreCache.ExternalPackagesCompatibilityChecker.ProcessCompatibility(List'1 packageVersions, Dictionary'2 resultsDict)`

Additionally I have no results in the API calls tab either.

**Comment by mhwkanon** [comment]
Hi @peteawood ,
Thanks for reporting this issue. Could you provide some information for our investigation?
1.  Have you enabled any http proxy when you running the assessment? 
2. What is your aws account id?

**Comment by ryankanno** [comment]
I'm also seeing similar behavior and have the exact same error messages appear in the assessment logs. (I don't have any http proxies enabled).

**Comment by peteawood** [comment]
Hi @mhwkanon, no, I have no proxy running. I have also recreated this issue on a fresh .Net Framework 4.7.2 solution with just a single Nuget dependency. I see the same errors in the log and the same behaviour (package status marked as Failed) in the tool.

**Comment by mhwkanon** [comment]
Thanks for reply and information @peteawood @ryankanno, we are sorry customers facing this issue. we will start the investigation and keep you updated.

**Comment by ryankanno** [comment]
Just to provide more clarity - I have a similar setup with a mix of public/private repositories without any http/https proxies enabled. With that said, I am also accessing the data store from Asia. I have enabled the data sharing options if you need any more info!

**Comment by mhwkanon** [comment]
@ryankanno thanks for the additional information, this definitely could help us to identify the root cause. We are investigating it now and will let you know once we can fix the issue.

**Comment by mrlloyd-amzn** [comment]
@ryankanno We just released 1.0.2 which should address the issue you are seeing. Please try it with the new release and let us know if you still see the issue. Closing this out for now.

**Comment by peteawood** [comment]
Just to confirm that with version 1.1.0 this problem appears resolved. Many thanks!
